config/server.ini

```diff
@@ -140,4 +140,8 @@ IgnorePacketFamilies = *
 ## InitLoginBan (bool)
 # Sends an INIT packet as a response when the login attempt comes from a banned user
 # When disabled, sends a LoginReply enum value instead
-InitLoginBan = yes
\ No newline at end of file
+InitLoginBan = yes
+
+## ThreadPoolThreads (int)
+# Number of background threads in the thread pool. A value of 0 defaults to the number of CPUs on the system
+ThreadPoolThreads = 0
```

Comment:
Since this is determined by `std::thread::hardware_concurrency`, my guess is that the default will correspond with the number of CPU cores (non-hyperthreaded) and logical cores (hyperthreaded).

> Returns the number of concurrent threads supported by the implementation. The value should be considered only a hint.

[Reference](https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency)

Reply:
Good point, I'll update the comment. I think I also need to handle if the value returned by this is zero.